File: dayOne/javascript/numberguessingapp.js
// collect number from the terminal using prompt-sync
// generate random number between 1 and 100

const prompt = require("prompt-sync")();

secretNum = Math.floor(Math.random() * 100 +1 );

count = 0;

while(true){
    count++;
    guessNum = Number(prompt("guess a number: "));
    
    
    if(guessNum > 100){
        console.log("engineer please abort!!!");
    }
    
     if(guessNum > secretNum){
         console.log("higher");
     } else if(guessNum < secretNum){
         console.log("lower");
     } else if(secretNum == guessNum){
         console.log("correct!!!");
         break;
     } 
     
     if(count == 5) {
         console.log("maximum tries exceeded");
         console.log(`${count} no of tries. Secrete number is ${secretNum}`)
         break;
     }
}
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
   // for(count = 1; count <= 5; count++){
   //     if(guessNum > secretNum){
   //         console.log("higher")
   //     } else if(guessNum < secretNum){
   //         console.log("lower")
   //     } else if(secretNum == guessNum){
   //         console.log("correct!!!")
   //     }
   // }

File: dayOne/java/NumberGuessingApp.java
public class NumberGuessingApp{
    public static void main(String[] args){
        
        java.util.Scanner inputCollector = new java.util.Scanner(System.in);
        java.util.Random random = new java.util.Random();
        
        
        int guessNum = inputCollector.nextInt();
        int secretNum = random.nextInt(100);
        // System.out.println(secretNum);
        
        
        int count = 0;
        
        while(true){
            System.out.println("Enter your number");
            guessNum = inputCollector.nextInt();
            count++;
            
            if(guessNum > 100){
                System.out.println("engineer please abort!!!");
            }
            
             if(guessNum > secretNum){
                 System.out.println("higher");
             } else if(guessNum < secretNum){
                 System.out.println("lower");
             } else if(secretNum == guessNum){
                 System.out.println("correct!!!");
                 break;
             } 
             
             if(count == 5) {
                 System.out.println("maximum tries exceeded");
                 System.out.println("%d no of tries".formatted(count));
                 System.out.println("secrete number is " + secretNum);
                 break;
             }
        }
    }
}
